package.json

```diff
@@ -53,6 +58,10 @@
     "@babel/preset-env": "^7.10.4",
     "@commitlint/cli": "^9.1.1",
     "@commitlint/config-conventional": "^9.1.1",
+    "@types/eslint": "^7.2.0",
+    "@types/fs-extra": "^9.0.1",
+    "@types/micromatch": "^4.0.1",
+    "@types/webpack": "^4.41.21",
```

Comment:
I think some of types (for public usage) should be in `dependencies`, otherwise you will get error